io.md

```diff
@@ -89,17 +89,15 @@ So far, when we have been writing and using strings, or reading and writing to a
 <class 'str'>
 ```
 
-When we read or write to a file or when we talk to other computers on the Internet, we need to convert our unicode strings into a format that can be sent and received, and that format is called "UTF-8". We can read and write in that format, using a simple keyword argument to our standard `open` function:
+The rules for translating Unicode into bytes (which is what computers use to process info) is called encoding.  Encoding is important when you want to send and receive info from other computers, and a popular encoding to use is UTF-8.  We can read and write in UTF-8 by using a simple keyword argument in our `open function`.
```

Comment:
I'm not sure this makes it clearer. At this point, does the reader understand when Unicode is used and when bytes are used?

Reply:
Well, I always appreciate things to be explained in as much reasonable detail myself, so I'm generally for the reader getting more information.  Here, I was just trying to define the term "encoding" and what it's used for as (when I first read your book) I couldn't tell you either.  Nor did I know that what Unicode was, haha.

How much more did you want to add here?  I'll take a stab at it, though I'm a little fuzzy on the details:
"Computers run on what is called machine code, which is comprised of bytes (which is just a group smaller units called bits) to represent data and instructions.  It's tedious for humans to program in machine code, so that is why we often use a higher-level language (something we can more easily read) like Python.  When a Python script executes, it translates the instructions written in Python into machine code so your computer can understand.  Some languages do more of this translation on the front-end (vs. waiting until run-time when the script is ran) through a process called compiling. 

When data is sent over the Internet, we need to send it in bytes & bits... something your computer easily understands.  The rules for translating Unicode (which is what Python uses when it stores a string) to bytes is called encoding.  A popular encoding to use is UTF-8.  We can read and write in UTF-8 by using a simple keyword argument in our `open function`."

Reply:
The second paragraph is useful, we can skip the first paragraph.